### ISSUE TITLE
feature: Fabric/Session management

### ISSUE THREAD

**Issue by kennylevinsen** [t0bst4r/home-assistant-matter-hub#226]
1. Once a bridge is in the commissioned state, it doesn't seem to allow commissioning additional fabrics to it, requiring multiple bridges to have both Apple Homekit and Google Home.

2. Once a fabric is added, the only option appears to be factory reset if it needs to be removed from the bridge side.

3. Session information for the fabrics would be useful.

**Comment by t0bst4r** [comment]
You can use the app of the already paired controller to set the bridge into pairing mode again.

https://github.com/t0bst4r/home-assistant-matter-hub/blob/main/packages/documentation/faq/Connect%20Multiple%20Fabrics.md

But yes, starting the pairing mode from the UI would be useful. As well as deleting fabrics.

**Comment by kennylevinsen** [comment]
> You can use the app of the already paired controller to set the bridge into pairing mode again.

Ah, you even had it document! Not the direction I was expecting the flow to be.

For Google Home, the flow ended up being to select the bridge, go to Linked Matter apps and services, Link apps and services, and then Share with QR Code:

![image](https://github.com/user-attachments/assets/82a4fc3b-e329-46f1-9add-4103d8aa953d)
![image](https://github.com/user-attachments/assets/27023328-88b4-44d3-b67b-fa5d3e226b86)
![image](https://github.com/user-attachments/assets/89021849-c98c-4c8b-97f0-b225be3f2eb1)

Keeping it open for the bridge-side management.

**Comment by Moonbase59** [comment]
> But yes, starting the pairing mode from the UI would be useful. As well as deleting fabrics.

Indeed, this affects me, too. I plan to add _both_ Alexa and Google Home to the same bridge. Is that even possible, because after the first fabric it shows "Commissioned" above the QR code?

**Comment by kennylevinsen** [comment]
> Indeed, this affects me, too. I plan to add _both_ Alexa and Google Home to the same bridge. Is that even possible, because after the first fabric it shows "Commissioned" above the QR code?

https://github.com/t0bst4r/home-assistant-matter-hub/blob/main/packages/documentation/faq/Connect%20Multiple%20Fabrics.md
